backend/src/routes/sessions.ts

```diff
@@ -51,19 +53,24 @@ router.post(
       }
       const accessToken = mentor.personalAccessToken;
       const data = await getCalendlyEventDate(req.body.calendlyURI, accessToken);
+      const startDate = new Date(2023, 0o5, 0o5, 17, 0, 0, 0);
+      const endDate = new Date(2023, 0o5, 0o5, 18, 0, 0, 0);
```

Comment:
What's the purpose of the startDate and endDate fields? Is it to ensure that startTime and endTime exists? Because I was thinking of calendly doesn't return the start and end time of the event it makes sense to just return an error rather than filling in some random event time.